backends/advanced/src/advanced_omi_backend/workers/orchestrator/process_manager.py

```
@@ -0,0 +1,296 @@
+        Returns:
+            True if started successfully, False otherwise
+        """
+        if self.process and self.is_alive:
+            logger.warning(f"{self.name}: Already running (PID {self.pid})")
+            return False
+
+        try:
+            logger.info(f"{self.name}: Starting worker...")
+            logger.debug(f"{self.name}: Command: {' '.join(self.definition.command)}")
+
+            self.process = subprocess.Popen(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)